dev/io.openliberty.checkpoint/src/io/openliberty/checkpoint/internal/criu/CheckpointFailedException.java

```diff
@@ -36,6 +36,7 @@ public enum Type {
         UNKNOWN_CHECKPOINT(75, false),
 
         JVM_RESTORE_FAILED(76, true),
+        SYSTEM_RESTORE_FAILED(79, true),
```

Comment:
I would put this before `JVM_RESTORE_FAILED(76, true),` and make `SYSTEM_RESTORE_FAILED`  use 76 return code.  Shift down all the rest:

```java
        SYSTEM_RESTORE_FAILED(76, true),
        JVM_RESTORE_FAILED(77, true),
        LIBERTY_RESTORE_FAILED(78, true),
        UNKNOWN_RESTORE(79, true);
```

Maybe best to move all the restore fails to start at error code 80 to give us a gap between the checkpoint failures (which start at 70) and the restore failures:

```java
        SYSTEM_RESTORE_FAILED(80, true),
        JVM_RESTORE_FAILED(81, true),
        LIBERTY_RESTORE_FAILED(82, true),
        UNKNOWN_RESTORE(83, true);
```
I'm not sure how many tests that could cause to fail, but we can adjust them.